### PR TITLE
Problem: no aggregated TX history command(#285)

### PR DESCRIPTION
**Pull request by linfeng-crypto**
add a new query command `txs-all` to query all the events transactions

### CODE REVIEW

**Review by tomtau — CHANGES_REQUESTED** [review]
a few linting issues: https://github.com/crypto-org-chain/chain-main/pull/433/checks?check_run_id=2199413268#step:4:77

**Comment by JayT106** [comment]
any test cases to cover the query?

File: x/chainmain/client/cli/query.go [hunk 1]
```diff
@@ -35,6 +37,52 @@ func GetQueryCmd() *cobra.Command {
...
+				return queryErr
+			}
+			addrStr := args[0]
+			_, addressErr := sdk.AccAddressFromBech32(addrStr)
+			if addressErr != nil {
+				return addressErr
+			}
+
+			var tmEvents []string
+			events := [...]string{"message.sender", "transfer.recipient", "withdraw_rewards.validator", "ibc_transfer.receiver"}
```
Comment (tomtau) [inline x/chainmain/client/cli/query.go:58]: not sure `ibc_transfer.sender` is also valid?

Comment (linfeng-crypto) [reply x/chainmain/client/cli/query.go:58]: I'll test

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: x/chainmain/client/cli/query.go (hunk 1 above)
Comment (tomtau) [inline x/chainmain/client/cli/query.go:58]:
```suggestion
			events := [...]string{"message.sender", "transfer.recipient", "withdraw_rewards.validator", "ibc_transfer.receiver", "ibc_transfer.sender"}
```

**Review by tomtau — APPROVED** [review]

**Pull request merged**